### ISSUE TITLE
Create a test-set and validation-set using the sklearn package

### ISSUE THREAD

**Issue by mrcolo** [datares/comma_ai#6]
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

**Comment by BruceLiu-dot** [comment]
I assume it's a typo in the title but just to clarify, we are creating train and validation sets, not a test set right? Don't we already have a test set? My question is should we randomly split the train to create the train and validation sets? In run.py, there is a comment saying that the order of the data matters so we shouldn't shuffle the data when passing to DataLoader. But if we create the training sets randomly, it would be as if the data was shuffled. 
There is another comment right below saying that we should shuffle the data but "pick batches of image that are related to themselves". I'm not sure if this is to be done within the process of splitting? On the other hand, I also don't know the definition of images relating to themselves.  
![Capture](https://user-images.githubusercontent.com/56210593/73909515-afff6780-4861-11ea-808a-68daa84f625c.PNG)

**Comment by mrcolo** [comment]
Yes, my bad!

**Comment by mrcolo** [comment]
You should not shuffle randomly, that should be handled by your dataloader class. In regards to the images, I was referring to use a sequence of images as input, so that the network can learn a more recurrent relationship.
